src/components/shared/TractionMenu.vue

```diff
@@ -1,11 +1,33 @@
 <template>
-  <menu class="list-none inline-flex ml-4 m-0 p-0 space-x-2 font-light">
+  <menu
```

Comment:
I think this is the right place but would it be possible to style this as per the new reception but with rounded corners? 
![Screenshot 2022-08-03 at 09 13 04](https://user-images.githubusercontent.com/519327/182558555-fcdb5035-a46d-4673-81be-0ac970acfac0.png)

Reply:
Also when you hover over the menu item it is white.

Reply:
Will do - rounded corners and hovering fix

Reply:
I can see the it is already rounded in mine and I have fixed the hovering issue

https://user-images.githubusercontent.com/55585488/183088856-b1e22af4-d4cd-4181-847b-ca87849251c0.mov